src/axislike.jl

```diff
@@ -120,7 +120,7 @@ function print_tex(io::IO, groupplot::GroupPlot)
             elseif elt isa Plot
                 print_tex(io, elt)
             elseif elt isa Void
-                print(io, raw"\nextgroupplot[group/empty plot]")
+                println(io, raw"\nextgroupplot[group/empty plot,xmin=0,xmax=1,ymin=0,ymax=1]")
```

Comment:
IMO a fundamental design principle of PGFPlotsX is to mirror LaTeX code very closely, so that examples found on the net can be easily applied, and, going in the other direction, `print_tex` can be used for debugging and comparison.

Introducing extra options just to suppress some warnings goes against this. IMO the preferred way to emit the above would be a 
```julia
@pgf { "group/empty plot", xmin = 0, xmax = 1, ymin = 0, ymax = 1}
```
within a `GroupPlot`.

Reply:
So then we should completely remove the `nothing` pun since that is not what you write in LaTex and for `@pfg{ "group/empty plot"}` everywhere?

The point of `nothing` is clearly to not produce a plot there, might as well do it without getting warnings from it.